azkaban-web-server/src/web/js/azkaban/view/job-list.js

```diff
@@ -43,6 +44,20 @@ azkaban.JobListView = Backbone.View.extend({
     this.list = $(this.el).find("#joblist");
     this.contextMenu = settings.contextMenuCallback;
     this.listNodes = {};
+
+    this.presetToggleBtn("autoPanZoom");
+    this.presetToggleBtn("autoExpand");
+  },
+
+  presetToggleBtn: function(name) {
+    var target = "#" + name + "Btn";
+    if (localStorage.getItem("LiAz-" + name) !== 'false') {
+      if ($(target).hasClass('btn-default')) $(target).removeClass('btn-default');
```

Comment:
Not sure if you feel like this should be extracted or not for similar logic.

Reply:
If we were to require it a second time, probably.
For now, it should be fine.